src/main/ui/chat/ChatChoices.tscn

```diff
@@ -47,4 +47,3 @@ bus = "Reverb Bus"
 stream = ExtResource( 5 )
 volume_db = -6.0
 bus = "Reverb Bus"
-[connection signal="gui_input" from="." to="." method="_on_gui_input"]
```

Comment:
This is worth mentioning in the commit comment; `resolved error related to invalid _on_gui_input connection in ChatChoices.tscn`

Reply:
Commit comment now mentions `Fixed error related to invalid _on_gui_input connection in ChatChoices.tscn`